src/test/java/com/coravy/hudson/plugins/github/GithubLinkAnnotatorTest.java

```diff
@@ -25,13 +44,44 @@ public final void testAnnotateStringMarkupText() {
                         + "issues/9876/find'>close #9876</a> link");
     }
 
+    @Test
+    public final void testChangeSetAnnotateStringMarkupText() {
```

Comment:
did you know about "Matchers" pattern?

code can be organized in form:
```java
assertThat(originalText, annotatedWith(changeSet, equalTo(expectedChangeSetAnnotation)));
```

Please try to use 
```java
public static Matcher<String> annotatedWith(GitChangeSet chSet, Matcher<String> matcher) {
   return new FeatureMatcher<String, String>(matcher, "annotated text", "") {
    
      @Override
       public String featureValueOf(String actual) {
            return annotate(actual, chSet);
       }
}
```

More info: http://hamcrest.org/JavaHamcrest/

Please also organize multiply cases same as in:
https://github.com/jenkinsci/github-plugin/blob/master/src/test/java/org/jenkinsci/plugins/github/util/XSSApiTest.java
More info: https://github.com/TNG/junit-dataprovider